src/property-filter/__integ__/async-loading.test.ts

```diff
@@ -206,3 +209,24 @@ test.each<TestCase>(testCases)('%p', (_, asyncProperties, token, scenario) =>
     }
   })()
 );
+
+test(
+  'calls onLoadItems with an empty string if Clear button is clicked',
+  setupTest(true, 'property', async page => {
+    await page.typeInFilteringInput('1');
+    await page.expectLoadItemsEvents([
+      { firstPage: true, samePage: false, filteringText: '' },
+      { filteringText: '1', firstPage: true, samePage: false },
+    ]);
+
+    await page.click('#focus-target');
+    await page.clearFilteringInput();
+    await page.waitForJsTimers(500);
+    await page.expectLoadItemsEvents([
+      { firstPage: true, samePage: false, filteringText: '' },
+      { filteringText: '1', firstPage: true, samePage: false },
```

Comment:
Why is there an event with filteringText: "1" following one with filteringText: ""?